### PR TITLE
Semesterify apps & projects

### PR DESCRIPTION
**Pull request by kevinshieh**

### CODE REVIEW

File: app/models/semester.rb [hunk 1]
```diff
@@ -23,6 +23,10 @@ def can_be_destroyed?
     ! (student_applications.exists? || projects.exists?)
   end
 
+  def self.current_semester
+    Semester.where("is_current_semester = 'true'").first
+  end
```
Comment (SamLau95) [inline app/models/semester.rb]: This can be simplified to just `where(...`

File: app/models/semester.rb [hunk 2]
```diff
@@ -23,6 +23,10 @@ def can_be_destroyed?
...
+
   def self.clear_current_semester
     Semester.where('is_current_semester').update_all("is_current_semester = 'false'")
```
Comment (SamLau95) [inline app/models/semester.rb]: This too

**Comment by SamLau95** [comment]
Just two very small changes, but otherwise good job!

File: app/models/semester.rb [hunk 3]
```diff
@@ -23,7 +23,11 @@ def can_be_destroyed?
     ! (student_applications.exists? || projects.exists?)
   end
 
+  def self.current_semester
+    where("is_current_semester = 'true'").first
+  end
+
   def self.clear_current_semester
-    Semester.where('is_current_semester').update_all("is_current_semester = 'false'")
+    where('is_current_semester').update_all("is_current_semester = 'false'")
```
Comment (negativetwelve) [inline app/models/semester.rb]: just curious, why are you guys using strings for where clauses?

Comment (kevinshieh) [inline app/models/semester.rb]: i saw it once on SO but i now realize is bad. and apparently vulnerable to sql injection

Comment (kevinshieh) [inline app/models/semester.rb]: changed in 0e63a58f06fe8d0eae2a2ff8687992efd4ce64dc

**Pull request merged**